### ISSUE TITLE
what are your standard data transforms -- especially for data augmentation?

### ISSUE THREAD

**Issue by brando90** [camillegontier/DELAUNAY_dataset#3]
is this really all the transforms you apply?
```
    train_dataset = ImageFolder(path2train, transform=transforms.Compose([
        transforms.Resize((size, size)),
        transforms.ToTensor(),
        transforms.Normalize(mean=mean,
                             std=std),
    ]))
```
@camillegontier 

Thanks! :)

**Comment by brando90** [comment]
what is the advantage of RandomCrop vs resize? 
```
    train_data_transforms = Compose([
        ToPILImage(),
        RandomCrop(84, padding=8),
        # todo: do we really need th padding = 8 for delauny, check notes, check l2l git issues if not post one
        ColorJitter(brightness=0.4, contrast=0.4, saturation=0.4),
        RandomHorizontalFlip(),
        ToTensor(),
        normalize,
    ])
    transforms.Compose([
            transforms.Resize((size, size)),
            transforms.ToTensor(),
            transforms.Normalize(mean=mean,
                                 std=std),
        ]))
```

which one to use?

**Comment by brando90** [comment]
discussion: https://datascience.stackexchange.com/questions/116201/when-to-use-padding-when-randomly-cropping-images-in-deep-learning

**Comment by brando90** [comment]
@camillegontier I will try this:
```
    train_data_transform = Compose([
        RandomResizedCrop((size, size), scale=scale, ratio=ratio),
        RandomCrop(size=size, padding=padding),
        ColorJitter(brightness=0.4, contrast=0.4, saturation=0.4),
        RandomHorizontalFlip(),
        ToTensor(),
        Normalize(mean=mean, std=std),
    ])
    test_data_transform = transforms.Compose([
        Resize((size, size)),
        ToTensor(),
        Normalize(mean=mean, std=std),
    ])
    validation_data_transform = test_data_transform
```

let me know if have feedback.
